nilearn/plotting/js_plotting_utils.py

```diff
@@ -196,13 +196,11 @@ def colorscale(cmap, values, threshold=None, symmetric_cmap=True, vmax=None):
                       'setting symmetric_cmap to True')
         symmetric_cmap = True
     if vmax is None:
-        if symmetric_cmap:
```

Comment:
But, aren't these routines compatible with other plotting functions ?

Reply:
I think that @KamalakerDadi and myself are getting to the same point there: we need to make this as much consistent as possible as possible across plotting functions, maybe by sharing this outside of the js plotting file.

Reply:
agreed